lib/entity/pin/PinTransactieMatch.php

```diff
@@ -223,4 +263,63 @@ public function getVerschil() {
 			return null;
 		}
 	}
+
+	/**
+	 * @return string
+	 */
+	public function gokStatus() {
```

Comment:
Ik mis een `rand` aanroep in deze gok functie 😛. Ik zou hier iets van `getDefaultStatus` ofzo doen.

Reply:
Enigszins geïnspireerd door [`guessExtension`](https://github.com/symfony/http-foundation/blob/1b66b861f3ff55fb39c13822a0fc915018b5ac84/File/File.php#L55) van Symfony, maar goed punt 😁